### PR TITLE
 Consider exotic but legal places for comments and spaces

### PR DESCRIPTION
**Pull request by mlangc**
This PR contains a fix for [#1002611](https://www.assembla.com/spaces/scala-ide/tickets/1002611-rename-refactoring-breaks-when-default-arguments-and-comments-spaces-are-involved/details#), as well as some mostly cosmetic changes.

### CODE REVIEW

File: src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala [hunk 1]
```diff
@@ -28,6 +28,8 @@ import scala.tools.refactoring.util.UniqueNames
 
 object TestHelper {
   case class PrepResultWithChanges(prepResult: Option[Either[MultiStageRefactoring#PreparationError, Rename#PreparationResult]], changes: List[Change])
+
+  private val PtnIndentedLine = """(\s*)[^\s].*""".r
```
Comment (sschaef) [inline src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala]: What does `Ptn` stand for?

Comment (mlangc) [inline src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala]: It stands for `Pattern`.. would you be more happy with `Rx` (for `Regex`)?

Comment (sschaef) [inline src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala]: I would use `IndentedLine. It is obvious that it is a regex, isn't it?

Comment (mlangc) [inline src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala]: You have a point... I've just renamed it.

**Pull request merged**